userAPI.js

```diff
@@ -0,0 +1,62 @@
+const Joi = require('joi');
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+const users = [
+    {id: 1, name: "user1", email: "user1@mail.ru"},
+    {id: 2, name: "user2", email: "user2@gmail.com"},
+    {id: 3, name: "user3", email: "user3@bk.ru"},
+
+]
+
+app.get('/api/users', function (req, res) {
+    res.send(users);
+});
+
+app.get('/api/users/:id', function (req, res) {
+    const user = users.find(us => us.id === parseInt(req.params.id));
+
+    if(!user) return res.status(404).send('The user with the given ID was not found');
```

Comment:
The error should also be a JSON if you use some format for your response, you should keep it in all endpoints, so your API will be predictable for others and easy to use. Let me know if it's not clear for you